pkgs/development/python-modules/clickclick/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub, flake8, click, pyyaml, six, pytest }:
+
+buildPythonPackage rec {
+  pname = "clickclick";
+  version = "1.2.2";
+
+  src = fetchFromGitHub {
```

Comment:
Can you add a comment that we're using `fetchFromGitHub` because tests weren't distributed on PyPi?
Also do that everywhere else where this is done.

Reply:
I've added a comment.